.github/workflows/CI.yml

```diff
@@ -27,8 +28,18 @@ jobs:
           sudo apt-get update
           sudo apt-get install mpich libhdf5-mpich-dev
           echo "JULIA_HDF5_PATH=/usr/lib/x86_64-linux-gnu/hdf5/mpich/" >> $GITHUB_ENV
-          echo "JULIA_MPI_BINARY=system" >> $GITHUB_ENV
       - uses: actions/checkout@v3
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+          show-versioninfo: true
```

Comment:
There is already a `julia-actions/setup-julia` job below. Please use it only once

Reply:
Done.